composer.json

```diff
@@ -35,5 +34,10 @@
     },
     "bin": [
         "surf"
-    ]
+    ],
+    "config": {
+        "platform": {
+            "php": "7.0.33"
```

Comment:
why?

Reply:
Because this prevents locking dependencies that require higher PHP versions during `composer update` if you have e.g. PHP 7.2 that executes composer.

Reply:
7.0.33 is the latest version in Debian oldstable https://packages.debian.org/stretch/php7.0-cli. I think we should not be more legacy than that.

Reply:
ok